types/priv_validator.go

```diff
@@ -105,13 +97,14 @@ func LoadPrivValidator(filePath string) *PrivValidator {
 	if err != nil {
 		Exit(err.Error())
 	}
-	privVal := wire.ReadJSON(&PrivValidator{}, privValJSONBytes, &err).(*PrivValidator)
+	privVal := PrivValidator{}
```

Comment:
I understand not wanting `privVal := PrivValidator{}`.  But the two solutions are functionally the same:

* `privVal := &PrivValidator{}`
* `privVal := new(PrivValidator)`

Do you want to enforce one style?  Or ask other devs what they prefer?

On a related note... Just did: `bres := (*tmResult).(*ctypes.ResultBroadcastTxCommit)` and cringed again